### PR TITLE
Updated rimraf version from 3.0.2 to 5.0.5

### PR DESCRIPTION
**Pull request by aman19K**
**CS-43934**

- [x] update rimraf version from 3.0.2 to 5.0.5. SRE issues
- [x] version bump already done

### CODE REVIEW

**Review by netrajpatel — APPROVED** [review]

File: packages/contentstack-clone/package.json [hunk 1]
```diff
@@ -5,22 +5,22 @@
...
     "inquirer": "8.2.4",
+    "lodash": "^4.17.20",
+    "merge": "^2.1.1",
     "ora": "^5.1.0",
     "prompt": "^1.3.0",
-    "rimraf": "^3.0.2",
-    "winston": "^3.7.2",
-    "merge": "^2.1.1",
-    "lodash": "^4.17.20"
+    "rimraf": "^5.0.5",
```
Comment (antonyagustine) [inline packages/contentstack-clone/package.json:22]: @aman19K As we've updated the MAJOR version, I hope it doesn't break any existing functionality.

**Review by antonyagustine — APPROVED** [review]

File: packages/contentstack-clone/package.json (hunk 1 above)
Comment (aman19K) [reply packages/contentstack-clone/package.json:22]: @antonyagustine ,This library is only being used for the cleanup step, which I took care of in clone.js. I'll ask Aniket to execute e2e before merging the code.

File: packages/contentstack-clone/package.json [hunk 2]
```diff
@@ -5,22 +5,22 @@
   "author": "Contentstack",
   "bugs": "https://github.com/rohitmishra209/cli-cm-clone/issues",
   "dependencies": {
+    "@colors/colors": "^1.5.0",
     "@contentstack/cli-cm-export": "~1.11.0",
     "@contentstack/cli-cm-import": "~1.14.0",
     "@contentstack/cli-command": "~1.2.16",
```
Comment (shafeeqd959) [inline packages/contentstack-clone/package.json:11]: Please do bump the version

Comment (aman19K) [reply packages/contentstack-clone/package.json:11]: @shafeeqd959 version bump already done from 1.10.0 to 1.10.1.

**Review by shafeeqd959 — APPROVED** [review]

**Review by cs-raj — APPROVED** [review]

**Pull request merged**